### DISCUSSION TITLE
post/2021-08-23-use-victoria-metrics-replace-prometheus/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#82] [Announcements]
# post/2021-08-23-use-victoria-metrics-replace-prometheus/

      本文讲述笔者将生产环境监控服务从 Prometheus 迁移到了 VictoriaMetrics, 并简单介绍其架构原理.
    

https://blog.cong.moe/post/2021-08-23-use-victoria-metrics-replace-prometheus/

**Comment by baojiliu** [comment]
hello，现在还在用吗？出现过什么重大问题吗
